src/Features/GrassLighting.cpp

```diff
@@ -48,10 +48,11 @@ void GrassLighting::DrawSettings()
 		if (ImGui::IsItemHovered()) {
 			ImGui::BeginTooltip();
 			ImGui::PushTextWrapPos(ImGui::GetFontSize() * 35.0f);
-			ImGui::Text("Subsurface Scattering (SSS) amount\n");
-			ImGui::Text("Soft lighting controls how evenly lit an object is.\n");
-			ImGui::Text("Back lighting illuminates the back face of an object.\n");
-			ImGui::Text("Combined to model the transport of light through the surface.");
+			ImGui::Text(
+				"Subsurface Scattering (SSS) amount "
```

Comment:
Add a . to the end.

Reply:
Fixed in 358ac33f14939cdc2eb5b4a2a33b9820ee888e2b